src/app/(protected)/(pages)/(dashboardPage)/create-page/_components/_Create-Page/RecentPrompts.tsx

```
@@ -0,0 +1,91 @@
+// Ensure Prompt type includes outlines
+type Prompt = {
+  id: string;
+  title: string;
+  createdAt: string | Date;
+  outlines: any[]; // outlines property is required for this component
+};
+import { motion } from "motion/react";
+import React from "react";
+import { toast } from "sonner";
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)